cumulusci/tasks/salesforce/BaseSalesforceApiTask.py

```diff
@@ -28,6 +28,9 @@ def _init_api(self, base_url=None):
         )
         if base_url is not None:
             rv.base_url += base_url
+
+        rv.headers.setdefault(CALL_OPTS_HEADER_KEY, self._get_client_name())
```

Comment:
Should be in the form `client=CLIENT_NAME`, not simply the CLIENT_NAME

Reply:
I woke up, hopped out of bed, and realized that this wasn't correct :)

Reply:
I don't know what that says about me.